### PR TITLE
✨ Raw results for best practices badge

### PR DESCRIPTION
**Pull request by laurentsimon**
Raw results for best practices badge
No breaking changes
```release-notes
Raw results for best practices badge
```

### CODE REVIEW

**Review by azeemshaikh38 — COMMENTED** [review]
Is this PR compiling?

File: checks/cii_best_practices.go [hunk 1]
```diff
@@ -40,31 +32,19 @@ func init() {
 	}
 }
 
-// CIIBestPractices runs CII-Best-Practices check.
+// CodeReview will check if the maintainers perform code review.
```
[the remark this answers is not in the document]
Comment (laurentsimon) [reply checks/cii_best_practices.go:35]: done.

**Comment by laurentsimon** [comment]
> Is this PR compiling?

it used to before a rebase. Fixed now.

**Comment by laurentsimon** [comment]
@azeemshaikh38 PTAL

**Review by azeemshaikh38 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**